pre-processor-scripts/populate-search-fields.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable */
+/* Since mongo does not support ES6 we can not lint with our current eslint config */
```

Comment:
Is this statement true? This would suggest ES6 implementation was added in `3.2` https://docs.mongodb.com/manual/release-notes/3.2-javascript/#modernized-javascript-implementation-es6